### PR TITLE
wv-2173 update MLS version

### PR DESCRIPTION
**Pull request by minniewong**
Update layer description references to version 5

## Description

Fixes WV-2173

- Updated layer description references to point to version 5 dataset landing page. 
- Fixed typo for CarbonMonoxide.md.

### CODE REVIEW

File: config/default/common/config/metadata/layers/mls/MLS_HNO3_46hPa_Night.md [hunk 1]
```diff
@@ -2,4 +2,5 @@ The Microwave Limb Sounder (MLS) Nitric Acid (46hPa, Night) layer shows nitric a
 
 The MLS Nitric Acid (HNO3) Mixing Ratio at 46hPa layer is derived from the MLS Nitric Acid  (ML2HNO3_NRT) available from the Microwave Limb Sounder (MLS) instrument on the Aura satellite. The sensor resolution is 5 km, imagery resolution is 2 km, and the temporal resolution is daily.
 
-References: [ML2HNO3_NRT](https://disc.gsfc.nasa.gov/datasets/ML2HNO3_NRT_004/summary)
\ No newline at end of file
+References: [ML2HNO3_NRT](https://disc.gsfc.nasa.gov/datasets/ML2HNO3_NRT_005
+/summary)
```
Comment (jwhurley1) [inline config/default/common/config/metadata/layers/mls/MLS_HNO3_46hPa_Night.md:6]: There is a break in the URL here which breaks the hyperlink. Just remove that and it should be good.

Comment (minniewong) [reply config/default/common/config/metadata/layers/mls/MLS_HNO3_46hPa_Night.md:6]: ack. i thought i had fixed it, but i guess i forgot to push the change.

**Review by jwhurley1 — CHANGES_REQUESTED** [review]
Just a small change in `config/default/common/config/metadata/layers/mls/MLS_HNO3_46hPa_Night.md`.  Need to remove the extra break in the URL.

**Review by jwhurley1 — APPROVED** [review]
Looks good!

**Pull request merged**